scripts/foundry/utils/Utils.s.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.17;
+
+import "forge-std/Script.sol";
```

Comment:
Actually we don't need this file it was only useful for the specific repo

Reply:
Removed